tools/docker/cpu_tests.Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG USE_BAZEL_VERSION=3.7.2
 
 RUN pip install --default-timeout=1000 tensorflow-cpu==$TF_VERSION
 
-RUN apt-get update && apt-get install -y sudo rsync cmake
+RUN apt-get update && apt-get install -y sudo rsync cmake libbz2-dev liblz4-dev libzstd-dev
```

Comment:
These dependencies have been installed to the basic CI docker.
https://github.com/tensorflow/recommenders-addons/blob/master/tools/docker/cuda11.2-cudnn8.1-ubuntu18.04-manylinux2010-multipython.Dockerfile#L30

Reply:
thanks

Reply:
Will remove the my side then.